WordPress/src/test/java/org/wordpress/android/ui/stories/SaveStoryGutenbergBlockUseCaseTest.kt

```diff
@@ -0,0 +1,288 @@
+package org.wordpress.android.ui.stories
+
+import android.content.Context
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.times
+import com.nhaarman.mockitokotlin2.verify
+import com.wordpress.stories.compose.story.StoryFrameItem
+import kotlinx.coroutines.InternalCoroutinesApi
+import org.assertj.core.api.Assertions
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.junit.MockitoJUnitRunner
+import org.wordpress.android.BaseUnitTest
+import org.wordpress.android.TEST_DISPATCHER
+import org.wordpress.android.fluxc.model.PostModel
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.fluxc.store.PostStore
+import org.wordpress.android.ui.posts.EditPostRepository
+import org.wordpress.android.ui.stories.SaveStoryGutenbergBlockUseCase.Companion.TEMPORARY_ID_PREFIX
+import org.wordpress.android.ui.stories.SaveStoryGutenbergBlockUseCase.StoryMediaFileData
+import org.wordpress.android.ui.stories.prefs.StoriesPrefs
+import org.wordpress.android.util.helpers.MediaFile
+
+@RunWith(MockitoJUnitRunner::class)
+class SaveStoryGutenbergBlockUseCaseTest : BaseUnitTest() {
+    private lateinit var saveStoryGutenbergBlockUseCase: SaveStoryGutenbergBlockUseCase
+    private lateinit var editPostRepository: EditPostRepository
+    private lateinit var siteModel: SiteModel
+    @Mock lateinit var storiesPrefs: StoriesPrefs
+    @Mock lateinit var context: Context
+    @Mock lateinit var postStore: PostStore
+
+    @InternalCoroutinesApi
+    @Before
+    fun setUp() {
+        saveStoryGutenbergBlockUseCase = SaveStoryGutenbergBlockUseCase(storiesPrefs)
+        siteModel = SiteModel()
```

Comment:
I think we can also mock this `siteModel` as well :) we can probably even remove it since it's just being used in one place.

Reply:
👍 done in 6b27610f4e thanks for the suggestion!